testsuite/single_node_performance.py

```diff
@@ -655,21 +668,30 @@ def print_table(
     print(calibrated_expected_tps)
 
     execute_command(f"cargo build {BUILD_FLAG} --package aptos-executor-benchmark")
-    print(f"Warmup - creating DB with {NUM_ACCOUNTS} accounts")
+    print(f"Warmup - creating a DB with {NUM_ACCOUNTS} default accounts")
     create_db_command = f"RUST_BACKTRACE=1 {BUILD_FOLDER}/aptos-executor-benchmark --block-executor-type aptos-vm-with-block-stm --block-size {MAX_BLOCK_SIZE} --execution-threads {NUMBER_OF_EXECUTION_THREADS} {DB_CONFIG_FLAGS} {DB_PRUNER_FLAGS} create-db {FEATURE_FLAGS} --data-dir {tmpdirname}/db --num-accounts {NUM_ACCOUNTS}"
     output = execute_command(create_db_command)
 
-    results = []
+    print(f"Warmup - creating another DB with {NUM_KEYLESS_ACCOUNTS} keyless accounts")
+    create_db_command_keyless = f"RUST_BACKTRACE=1 {BUILD_FOLDER}/aptos-executor-benchmark --block-executor-type aptos-vm-with-block-stm --block-size {MAX_BLOCK_SIZE_KEYLESS} --execution-threads {NUMBER_OF_EXECUTION_THREADS} {DB_CONFIG_FLAGS} {DB_PRUNER_FLAGS} --use-keyless-accounts create-db {FEATURE_FLAGS} --data-dir {keyless_tmp_dir}/db --num-accounts {NUM_KEYLESS_ACCOUNTS}"
```

Comment:
wondering if we can have a single db with both ? 

also - based on FLOW - keyless run might be requested or not needed. We don't need to warmup keyless if keyless is not detected. we can have bool - keyless_warmed_up, and first time we need keyless, initialize those accounts.

Reply:
> wondering if we can have a single db with both ?
it requires more refactoring of executor benchmark... better be a separate pr?

Reply:
maybe for now leave it as separate DB. but make the change to only create it if keyless test is being run

Reply:
fixed